pkg/api/octopus/mock.go

```diff
@@ -0,0 +1,52 @@
+package octopus
```

Comment:
Maybe this can go into the mocks package so they are not in the binary when building the app.